docs/contributing/index.rst

```diff
@@ -63,11 +63,23 @@ And now you can use gulp for installing views, by just running a simple command:
 
     $ yarn run gulp
 
-For the contributing process questions, please refer to the `Contributing Guide <http://docs.sylius.org/en/latest/contributing/index.html>`_ that comes up in the following chapters.
+For the contributing process questions, please refer to the `Contributing Guide <http://docs.sylius.org/en/latest/contributing/index.html>`_ that comes up in the following chapters:
+
+.. toctree::
+    :maxdepth: 1
+
+    code/index
+    documentation/index
+    translations/index
 
 .. _Gulp: http://gulpjs.com/
 .. _Node.js: https://nodejs.org/en/download/
 
+How to contribute using Sylius Vagrant
+--------------------------------------
+
+If you would like to contribute to the repo without worrying about environment, you can follow `this guide <https://github.com/Sylius/Vagrant/blob/master/README.md#configure-vagrant-to-contribute-on-sylius-core>`_ to setup your vagrant machine.
```

Comment:
This reference `https://github.com/Sylius/Vagrant/blob/master/README.md#configure-vagrant-to-contribute-on-sylius-core` is failing the build @loic425 :(

Can we do it this way?:
```
If you would like to contribute to the repo without worrying about environment, you can follow 
`the "Configure Vagrant to contribute on Sylius Core" section here <https://github.com/Sylius/Vagrant/blob/master/README.md>`_ 
to setup your vagrant machine.
```

Reply:
@CoderMaggie cool you found why this build fail. I'm gonna do that on monday.